cg/apps/housekeeper/hk.py

```diff
@@ -352,10 +348,7 @@ def add_and_include_file_to_latest_version(
         self, bundle_name: str, file: Path, tags: list
     ) -> None:
         """Adds and includes a file in the latest version of a bundle."""
-        version: Version = self.last_version(bundle_name)
-        if not version:
-            LOG.warning(f"Bundle: {bundle_name} not found in Housekeeper")
-            raise HousekeeperBundleVersionMissingError
+        version: Version = self.get_create_version(bundle_name)
```

Comment:
```suggestion
        version: Version = self.ensure_version(bundle_name)
```
Is this supposed to verify the existence of a "last_version" and make one if it doesn't exist? The `get_create_version` sounds a bit confusing to me.

Reply:
yes, that is exactly what it does

Reply:
I will rename to `get_or_create_version`

Reply:
sounds good, just as a note, in the storehelpers of our tests we use `ensure_case`, `ensure_customer` etc. for the `get_or_create` functionality. I don't mind either way.